superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -187,26 +213,7 @@ class ChartContainer extends React.Component {
             </div>
           }
         >
-          {this.props.alert &&
-            <Alert bsStyle="warning">
-              {this.props.alert}
-              <i
-                className="fa fa-close pull-right"
-                onClick={this.removeAlert.bind(this)}
-                style={{ cursor: 'pointer' }}
-              />
-            </Alert>
-          }
-          {this.props.isChartLoading ?
-            (<img alt="loading" width="25" src="/static/assets/images/loading.gif" />)
-            :
-            (<div
-              id={this.props.containerId}
-              ref={(ref) => { this.chartContainerRef = ref; }}
-              className={this.props.viz_type}
-              style={{ 'overflow-x': 'scroll' }}
-            />)
-          }
+          {this.renderChart(this.props.alert, this.props.isChartLoading)}
```

Comment:
maybe I'm missing something but you shouldn't have to pass props as they'll be in scope in the function